.gitignore

```diff
@@ -5,6 +5,7 @@
 
 # Credentials
 dbcred*
+service_account_key.json
```

Comment:
I'm not sure where we're gonna store the service account key, but we certainly shouldn't be saving it in the repo.